### PR TITLE
Binaries for automation client and server

### PR DESCRIPTION
**Pull request by michael-berlin**
@aaijazi

### CODE REVIEW

File: go/cmd/automation_client/automation_client.go [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+	fmt.Printf("Sending request:\n%v", proto.MarshalTextString(enqueueRequest))
+	enqueueResponse, err := client.EnqueueClusterOperation(context.Background(), enqueueRequest)
+	if err != nil {
+		fmt.Println("Failed to enqueue ClusterOperation. Error:", err)
+		os.Exit(4)
+	}
+	fmt.Println("Operation was enqueued. Details:", enqueueResponse)
+
+	finished := false
+wait_for_clusterop:
```
Comment (aaijazi) [inline go/cmd/automation_client/automation_client.go]: I think you can rewrite this from a loop label to a function call, which is a bit more readable to me.

**Comment by aaijazi** [comment]
LGTM, small comment, up to you if you want to address it.

**Comment by michael-berlin** [comment]
PTAL

I've moved it into a function.

**Comment by aaijazi** [comment]
Nice, looks much cleaner to me, LGTM

**Pull request merged**